### ISSUE TITLE
Add Hactoberfest label

### ISSUE THREAD

**Issue by bhantsi** [Vidit-jindal/Hacktoberfest-2021#9]
Please add hacktoberfest and hactoberfest-accepted to this repo or else all the contributions will not be counted

**Comment by rishiraj52** [comment]
@bhantsi  yes and my contributions are not counted on because of not having a hacktoberfest-accepted label in this repo.
